index.d.ts

```diff
@@ -464,8 +464,2262 @@ interface Typography {
 
 export interface Theme {}
 
-export const defaultTheme: Theme
-export const classicTheme: Theme
+export interface DefaultTheme extends Theme {
+  tokens: {
```

Comment:
@akleiner2 After digging into the code more, it looks like this `tokens` field is actually deprecated and we're trying to move away from it in favor of bringing all of the other properties (`colors`, `components`, etc) up a level. Wondering what would be the best course of action:

1. Leave it in, since it is currently available and consumers can tap into it
2. Leave it in but add a deprecation warning doc comment
3. Remove it from the type definition since we're trying to remove it from the object entirely